docs/cross-platform-targeting.md

```diff
@@ -64,6 +35,48 @@ Targeting .NET Standard, and successfully compiling your project, does not guara
 * [.NET Standard](https://docs.microsoft.com/en-us/dotnet/standard/net-standard)
 * [.NET API analyzer](https://docs.microsoft.com/en-us/dotnet/standard/analyzers/api-analyzer)
 
+## Multi-targeting
+
+An alternative way to add cross-platform support to a .NET library is multi-targeting. Multi-targeting involves targeting .NET implementations individually and includes multiple assemblies in a NuGet package. This leverages NuGet's ability to have multiple assemblies in a package and select the best one when added to an application.
+
+You can combine multi-targeting with .NET Standard. For example, a `netstandard2.0` target could be the default implementation and you could provide a .NET implementation specific target that uses implementation specific APIs for additional features. NuGet will automatically select the implementation specific target when possible, e.g. a .NET Core application will use the `netcoreapp2.0` assembly over the `netstandard2.0` assembly in a NuGet package.
+
+![NuGet package with multiple assemblies](../images/nuget-package-multiple-assemblies.png "NuGet package with multiple assemblies")
+
+**✔️ CONSIDER** targeting .NET implementations in addition to .NET Standard.
+
+> Targeting .NET implementations allows you to call platform-specific APIs that are outside of .NET Standard.
+>
+> Do not drop support for .NET Standard when you do this. Instead, throw from the implementation and offer capability APIs. This way, your library can be used anywhere and supports runtime light-up of features.
+
+**❌ DO NOT** use multi-targeting with .NET Standard if your source code is the same for all targets.
+
+> The .NET Standard assembly will automatically be used by NuGet. Targeting individual .NET implementations increases the `*.nupkg` size for no benefit.
+
+**✔️ CONSIDER** adding a target for `net461` when you're offering a `netstandard2.0` target. 
+
+> Using .NET Standard 2.0 from .NET Framework has some issues that were addressed in .NET Framework 4.7.2. You can improve the experience for developers that are still on .NET Framework 4.6.1 - 4.7.1 by offering them a binary that is built for .NET Framework 4.6.1.
+
+**✔️ DO** distribute your library using a NuGet package.
+
+> NuGet will select the best target for developer and shield them having to pick the appropriate implementation.
+
+**✔️ DO** use a project file's `TargetFrameworks` property when multi-targeting.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- This project will output netstandard2.0 and net471 assemblies -->
+    <TargetFrameworks>netstandard2.0;net471</TargetFrameworks>
+  </PropertyGroup>
+</Project>
+```
```

Comment:
We should add a plug for Oren's package here:

* **✔️ CONSIDER** using [MSBuild.Sdk.Extras](https://github.com/onovotny/MSBuildSdkExtras) when multi-targeting for UWP and Xamarin as it greatly simplifies your project file.